reference/connector.md

```diff
@@ -214,9 +214,8 @@ int wb_connector_get_presence(WbDeviceTag tag);
 **Description**
 
 The `wb_connector_enable_presence()` function starts querying the
-[Connector](#connector)'s *presence* (see definition below) state each `ms`
-milliseconds.
-The provided `ms` argument specifies the [Connector](#connector)'s sampling period.
+[Connector](#connector)'s *presence* (see definition below).
+The provided `sampling_period` argument specifies the sampling period of the [Connector](#connector) and is expressed in milliseconds.
```

Comment:
In this case it seems that some information are lost during the rephrasing.
I would explicitly write that the sampling period refers to the rate at which the presence of the connector is queried.